docs/PARAMETERS.md

```diff
@@ -11,17 +11,21 @@ Following options can be added to macro files, that are read by Geat4. Example f
  `/jpetmc/detector/loadTargetForRun runNr`  
 * shows only scintillator wrapped in a kapton foil:  
  `/jpetmc/detector/loadOnlyScintillators`  
-* not implemented option that should load n-layer fully packed with 
+* not implemented option that should load n-layer fully packed with
   scintillators barrel:  
  `/jpetmc/detector/loadIdealGeom`  
 * loads modular layer in two configurations (Single and Double)
  `/jpetmc/detector/loadModularLayer [option]`  
+* load detector configuration from JSON file
+ `/jpetmc/detector/jsonSetupFile [fileName]`
+* set the number of Run that is to be loaded from JSON file
```

Comment:
I wonder if we should drop the run number as for the past years we have always used the single setup description contained in the JSON file (though in principle the JSON file can contain more that 1 setup).
If so, we should eventually do that in the framework too, but here it would be a good point to start. What do you think about it?

Reply:
I was working with files that we have available, so including run number. I think that would require separate PRs, simultaneously to this project and Framework.

Reply:
Ok, let us leave the run number for now. I don't mean removing the run number from JSON files completely, just assuming that the file only contains a single setup and making the separate macro/command line parameter with run number optional.
However, you are right that this is a subject for a separate PR anyway.